docs/installation/index.md

```diff
@@ -1,7 +1,18 @@
 # Installation
 
-* Download and install Java 1.8, then set an environment variable: `export JAVA_HOME=$(/usr/libexec/java_home -v 1.8)`
-* Get Spark 2.3.x: [Download](https://spark.apache.org/downloads.html), unzip it and then set an environment variable: `export SPARK_HOME=<SPARK_FOLDER>`
+* Download and install Java 1.8
+    * To stay sane, some prefer to use JENV to manage different versions of Java: [Recommended Blog](https://medium.com/@danielnenkov/multiple-jdk-versions-on-mac-os-x-with-jenv-5ea5522ddc9b)
+    * On a Mac you might install Java 1.8 (on Mac) with: `brew cask install homebrew/cask-versions/zulu8`
```

Comment:
@rpanosSFDC this line repeats `Mac` twice

Reply:
[hyper-embarrassed face]

Thank you.